pkg/skaffold/kubernetes/portforward/resource_forwarder.go

```diff
@@ -67,14 +67,23 @@ func NewUserDefinedForwarder(entryManager *EntryManager, userDefinedResources []
 func (p *ResourceForwarder) Start(ctx context.Context, namespaces []string) error {
 	if len(namespaces) == 1 {
 		for _, pf := range p.userDefinedResources {
-			if pf.Namespace == "" {
+			if pf.Namespace != "" {
+				err := applyNamespaceWithTemplate(pf)
+				if err != nil {
```

Comment:
Minor nit: since you're not using `err` elsewhere you can inline this and save a line
```suggestion
				if err := applyNamespaceWithTemplate(pf); err != nil {
```

Reply:
@briandealwis can you approve?

Reply:
I'm a Smalltalker by background, and Smalltalk encourages small methods.  But separate functions/methods do impose a bit of cognitive overhead as it requires switching contexts when reading through the code to understand what it's doing.  In this case, the only real benefit is a shared text string, which isn't a big win.  And when I first looked at the function itself, I remember checking whether you were using it with other objects, particularly with live objects within the skaffold pipeline.

Anywys, I won't object to it.